src/main/java/com/joyent/manta/monitor/JMXMetricsCollector.java

```diff
@@ -42,6 +42,17 @@ public JMXMetricsCollector(final PlatformMbeanServerProvider platformMbeanServer
     public <T extends Number> T getMBeanAttributeValue(final String mBeanObjectName, final String attribute, final Class<T> returnType) {
         ObjectName objectName = getObjectNameFromString(mBeanObjectName);
         if (objectName == null) {
+            // This also indicates that the mbeanObject is not registered.
+            // Check to see if it is an exceptions-$class object
+            if (mBeanObjectName.startsWith("exceptions")) {
+                // The exceptions object is not added to the MBeanServer, as there
+                // were no HTTP client exceptions raised yet. Return 0.
+                if (returnType.equals(Double.class)) {
+                    return (T) new Double("0");
```

Comment:
This can be simplified to:
```java
return (T) Double.valueOf(0d);
```

This is preferred because the JVM may do optimizations for boxing/unboxing.